### PR TITLE
JS - update to node.js v12 quickstart

### PR DESCRIPTION
**Pull request by diberry**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by diberry** [comment]
@tamram @norman-v Please review and merge.

File: blobs/quickstarts/JavaScript/V12/nodejs/index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+const AZURE_STORAGE_CONNECTION_STRING =
+  process.env.AZURE_STORAGE_CONNECTION_STRING;
+
+if (!AZURE_STORAGE_CONNECTION_STRING) {
+  throw Error("Azure Storage Connection string not found");
+}
+
+async function main() {
+  console.log("Azure Blob storage v12 - JavaScript quickstart sample");
+  // Quick start code goes here
```
Comment (tamram) [inline blobs/quickstarts/JavaScript/V12/nodejs/index.js:14]: should we remove this comment - looks like it's from a template?

**Review by tamram — COMMENTED** [review]
Looks great - just one minor nit

File: blobs/quickstarts/JavaScript/V12/nodejs/index.js (hunk 1 above)
Comment (diberry) [reply blobs/quickstarts/JavaScript/V12/nodejs/index.js:14]: The `// Quick start code goes here` is meant to indicate where the reader of the quickstart is to copy each code section in. I can go either way with keep it or remove it.

Comment (diberry) [reply blobs/quickstarts/JavaScript/V12/nodejs/index.js]: @tamram Hi Tamra, I made the change, can you please merge?

**Pull request merged**